great_expectations/expectations/core/expect_column_quantile_values_to_be_between.py

```diff
@@ -131,16 +131,78 @@ class ExpectColumnQuantileValuesToBeBetween(ColumnExpectation):
     success_keys = (
         "quantile_ranges",
         "allow_relative_error",
+        "auto",
+        "profiler_config",
     )
     default_kwarg_values = {
         "row_condition": None,
-        "allow_relative_error": None,
         "condition_parser": None,
         "quantile_ranges": None,
         "result_format": "BASIC",
         "allow_relative_error": False,
```

Comment:
What's the significance of None->False?

Reply:
Hmmm, I *think* that the non-bool values were used by redshift, and it looks to me like below you've got it set to "linear"?

Reply:
@jcampbell Yes, I use "linear".  So should we change it back to `None` and not expect it to be strictly boolean?  Perhaps `Union[bool, str]`?  Also, do you mind if I delete the code regarding the unused variable?  Thanks.

Reply:
@jcampbell I made the change I proposed.  Thanks.

Reply:
@jcampbell Actually, this change causes downstream errors (Spark expects boolean).  I will change the type back.  Thanks.